tests/org.jboss.tools.freemarker.ui.bot.test/src/org/jboss/tools/freemarker/ui/bot/test/editor/OutlineLevelOfDetail.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
```

Comment:
Is this file even used? I can see you moved it, but I can see no updated references to this file.